src/QRadium/Version.cpp

```diff
@@ -0,0 +1,12 @@
+#include "Version.hpp"
```

Comment:
This file is auto-generated, so it shouldn't be included (actually it should be added to the .gitignore)